modules/extensions-mule-support/src/main/java/org/mule/runtime/module/extension/mule/internal/loader/parser/utils/AggregatedErrorsCharacteristic.java

```diff
@@ -0,0 +1,296 @@
+      List<ComponentAst> hierarchy = componentAstWithHierarchy.getHierarchy();
+      if (isRaiseError(operationAst)) {
+        handleRaiseError(operationAst, models, hierarchy);
+      } else {
+        handleOperationOtherThanRaiseError(operationAst, models, hierarchy);
+      }
+
+      return models;
+    }
+
+    private void handleOperationOtherThanRaiseError(ComponentAst operationAst, List<ErrorModelParser> models,
+                                                    List<ComponentAst> hierarchy) {
+      Optional<OperationModel> operationModel = operationAst.getModel(OperationModel.class);
+      operationModel.ifPresent(model -> model.getErrorModels().stream().map(MuleSdkErrorModelParser::new)
+          .forEach(errorModelParser -> addIfNotSuppressed(errorModelParser, models, hierarchy)));
+    }
+
+    private void handleRaiseError(ComponentAst raiseErrorAst, List<ErrorModelParser> errorModels, List<ComponentAst> hierarchy) {
+      final ComponentParameterAst typeParameter = raiseErrorAst.getParameter(DEFAULT_GROUP_NAME, ERROR_TYPE_PARAM);
+      if (null == typeParameter) {
+        return;
+      }
+
+      Optional<String> errorId = typeParameter.getValue().<String>getValue();
+      if (!errorId.isPresent()) {
+        return;
+      }
+
+      // TODO: Use the extension parser's namespace.
+      addIfNotSuppressed(new MuleSdkErrorModelParser(APP_LOCAL_EXTENSION_NAMESPACE, errorId.get(), null), errorModels, hierarchy);
+    }
+
+    private boolean isRaiseError(ComponentAst operationAst) {
+      return operationAst.getIdentifier().equals(RAISE_ERROR_IDENTIFIER);
+    }
+
+    private void addIfNotSuppressed(ErrorModelParser errorModelParser, List<ErrorModelParser> models,
+                                    List<ComponentAst> hierarchy) {
+      List<ErrorModelParserMatcher> suppressedErrors = getSuppressedErrors(hierarchy);
+      if (!isErrorSuppressed(errorModelParser, suppressedErrors)) {
+        models.add(errorModelParser);
+      }
+    }
+
+    private List<ErrorModelParserMatcher> getSuppressedErrors(List<ComponentAst> hierarchy) {
+      if (hierarchy.isEmpty()) {
+        return emptyList();
+      }
+
+      int containerIndex = hierarchy.size() - 1;
+      List<ErrorModelParserMatcher> suppressedErrors = new ArrayList<>();
+      while (containerIndex >= 0) {
+        ComponentAst container = hierarchy.get(containerIndex);
+        if (container.getIdentifier().equals(TRY_IDENTIFIER)) {
+          addSuppressedErrors(container, suppressedErrors);
+        } else if (container.getIdentifier().equals(ERROR_HANDLER_IDENTIFIER)) {
+          --containerIndex;
+        }
+        --containerIndex;
+      }
+      return suppressedErrors;
+    }
+
+    private void addSuppressedErrors(ComponentAst tryScopeAst, List<ErrorModelParserMatcher> suppressedErrors) {
+      Optional<ComponentAst> errorHandler = tryScopeAst.directChildrenStreamByIdentifier(null, ERROR_HANDLER).findFirst();
```

Comment:
Shouldn't we set `CORE_PREFIX` for the namespace matcher?
According to the Javadoc, `null` acts as a wildcard, meaning that will match anything.
What if there ever exists some component (from an extension most likely) named `<some-extension:error-handler>`? I guess we don't want to take any action on that one, because it wouldn't be related with the `try` scope.